src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -79,9 +83,51 @@ public async Task RunBatchAsync(QualificationRunOptions runOptions)
          _logger.AddInfo($"Qualification scenario finished in {timeSpent.ToDisplay()}");
       }
 
-      private Task<SaticFiles> copyStaticFiles(dynamic qualificationPlan)
+      private Task updateProjectsFullPath(IReadOnlyList<Project> projects) => Task.WhenAll(projects.Select(updateProjectFullPath));
+
+      private async Task<string> downloadRemoteFile(string url, string fileName, string subFolder, string id, string type)
       {
-         var staticFiles = new SaticFiles();
+         _logger.AddDebug($"Downloading {type.ToLower()} file for {id}...");
+         var downloadFolder = Path.Combine(_runOptions.TempFolder, subFolder);
+         DirectoryHelper.CreateDirectory(downloadFolder);
+
+         var fileFullPath = Path.Combine(downloadFolder, fileName);
+
+         using (var wc = new WebClient())
+         {
+            try
+            {
+               await wc.DownloadFileTaskAsync(url, fileFullPath);

```

Comment:
I will check if it works behind the firewall. If not: firewall proxy must be passed as an option to QualificationRunner and set into `wc.Proxy`

Reply:
I'd suggest to open another issue for that.

Reply:
yep